cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaConnectClusterTest.java

```diff
@@ -116,9 +122,29 @@
             .build();
     private final KafkaConnectCluster kc = KafkaConnectCluster.fromCrd(resource, VERSIONS);
 
+    @Deprecated
+    @Test
+    public void testMetricsConfigMapDeprecatedMetrics() {
+        KafkaConnect resource = new KafkaConnectBuilder(ResourceUtils.createEmptyKafkaConnect(namespace, cluster))
+                .withNewSpec()
+                .withMetrics((Map<String, Object>) TestUtils.fromJson(metricsCmJson, Map.class))
+                .withMetricsConfig(null)
+                .withConfig((Map<String, Object>) TestUtils.fromJson(configurationJson, Map.class))
+                .withImage(image)
+                .withReplicas(replicas)
+                .withReadinessProbe(new Probe(healthDelay, healthTimeout))
+                .withLivenessProbe(new Probe(healthDelay, healthTimeout))
+                .withBootstrapServers(bootstrapServers)
+                .endSpec()
```

Comment:
Just a nit ... but maybe you can try to fix the alignment to make it more readable.